static/src/javascripts/projects/common/views/experiments/participation/low-friction-buttons.html

```diff
@@ -0,0 +1,6 @@
+<<%=elType%>
```

Comment:
What do you think of having a single template, with the different permutations in if-blocks? I wonder if it might tweaking and rapidly iterating a little quicker. I guess it could also grow unwieldy quite quickly, too, so YMMV.